app/routes/goal_routes.py

```diff
@@ -0,0 +1,105 @@
+from flask import Blueprint, request, make_response, jsonify, abort
+from flask.wrappers import Response
+from app.models.goal import Goal
+from app.models.task import Task
+from app import db
+from datetime import date, datetime
+
+goals_bp = Blueprint('goals', __name__, url_prefix='/goals')
+
+# # Helper functions
+def valid_int(id):
```

Comment:
Note that you have this function in goal_routes and task_routes. Consider making a separate file with shared helper functions and importing them where you need them to DRY up your code.